recipe/migrations/arch_rebuild.txt

```diff
@@ -696,6 +698,3 @@ lammps
 phonopy
 pyscal
 pygplates
-libcublas
-libcublas-dev
-libcublas-static
```

Comment:
More details in comment ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4306#discussion_r1162101250 )